next.clayui.com/content/docs/css/components/popovers-and-tooltips.md

```diff
@@ -0,0 +1,469 @@
+---
+title: "Popovers and Tooltips"
+description: "Popovers and Tooltips are patterns used to show helpful information in two different ways."
+---
+
+<div class="nav-toc">
+
+- [Popover](#popover)
```

Comment:
Rather than repeat "Popover" here, I think it makes more sense to do something like

```md
- [Popover](#popover)
    - [Position](#popover-position)
        - [Top](#popover-top)
        - [Right](#popover-right)
        - [Bottom](#popover-bottom)
        - [Left](#popover-left)
```

Same for tooltip section, and same changes for the headings below.

Reply:
I didn't know that it will works but worked well! Thanks for the tip Bryce.